### PR TITLE
fix: Scrolling in Edit Profile

### LINKED ISSUE #2050: bug: Scrolling in Edit profile
**Describe the bug**
While Scrolling the Edit Profile layout in landscape mode the content hides behind the Done buttoon

**To Reproduce**
1. Go to Edit Profile
2. Change to landscape mode
3. Scroll down to bottom

**Expected behavior**
The date should scroll above  the button

**Screenshots**
bug_edit_profile_scroll

**Smartphone Info:**

|               | |
|---------------|-|
|Device         |Redmi Note 4|
|Android Version|Nougat 7.0|

**Would you like to work on the issue?**

- [x] Yes
- [ ] No
- Other:

### PR DESCRIPTION
**Pull request by dheerajkotwani**
Fixes #2050  Scrolling in Edit Profile Fragment

Checklist:

- [x] I have checked for PMD and check-style issues 
- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [x] My branch is up-to-date with the Upstream development branch.

Changes: 
Made the scrolling properly in Edit Profile Fragment

Screenshots for the change:
fix_scroll

### CODE REVIEW

**Comment by dheerajkotwani** [comment]
@iamareebjamal please review this.

File: app/src/main/res/layout/update_organizer_layout.xml [hunk 1]
```diff
@@ -37,14 +37,15 @@
 
             </com.google.android.material.appbar.AppBarLayout>
 
-            <androidx.coordinatorlayout.widget.CoordinatorLayout
+            <RelativeLayout
```
Comment (iamareebjamal) [inline app/src/main/res/layout/update_organizer_layout.xml]: LinearLayout

Comment (dheerajkotwani) [reply app/src/main/res/layout/update_organizer_layout.xml]: Done

File: app/src/main/res/layout/update_organizer_layout.xml [hunk 2]
```diff
@@ -59,22 +61,23 @@
                     android:layout_height="@dimen/submit_button_width"
                     android:text="@string/done"/>
 
-                <FrameLayout
-                    android:layout_width="@dimen/progressbar_large"
-                    android:layout_height="@dimen/progressbar_large"
-                    android:layout_gravity="center">
+            </LinearLayout>
+        </LinearLayout>
```
Comment (iamareebjamal) [inline app/src/main/res/layout/update_organizer_layout.xml:65]: Why's this needed?

Comment (dheerajkotwani) [reply app/src/main/res/layout/update_organizer_layout.xml:65]: When layout was inside the linear layout, as when the progress bar will get its visibility being at end of screen it remain out of the screen. So I moved the Frame Layout of Progress bar as when it is visible it come above the linear layout.

Comment (991rajat) [reply app/src/main/res/layout/update_organizer_layout.xml:65]: Add margin-bottom of button width in the nested scroll view as you can see in different edit layout's present in project structure.

Comment (dheerajkotwani) [reply app/src/main/res/layout/update_organizer_layout.xml:65]: defining a margin bottom can create problem in future as if design or height of button is changed then we will also require to change the margin accordingly

File: app/src/main/res/layout/update_organizer_layout.xml [hunk 3]
```diff
@@ -37,14 +37,16 @@
...
-            <androidx.coordinatorlayout.widget.CoordinatorLayout
+            <LinearLayout
                 android:layout_width="match_parent"
                 android:layout_height="match_parent"
+                android:orientation="vertical"
                 android:background="@color/color_top_surface">
 
                 <androidx.core.widget.NestedScrollView
                     android:layout_width="match_parent"
-                    android:layout_height="match_parent">
```
Comment (991rajat) [inline app/src/main/res/layout/update_organizer_layout.xml:47]: Add margin-bottom here. Dimension is defined in dimen.xml for submit button.

**Pull request closed without merging**